src/webgl/shaders/lighting.glsl

```diff
@@ -111,7 +111,7 @@ vec3 calculateImageDiffuse( vec3 vNormal, vec3 vViewPosition ){
   vec3 worldCameraPosition =  vec3(0.0, 0.0, 0.0);  // hardcoded world camera position
   vec3 worldNormal = normalize(vNormal);
   vec2 newTexCoor = mapTextureToNormal( worldNormal );
-  vec4 texture = TEXTURE( environmentMapDiffused, newTexCoor );
+  vec4 texture = textureCube( environmentMapDiffusedCubemap, newTexCoor );
```

Comment:
Previously, we were using the `TEXTURE`, which refers to a macro in https://github.com/processing/p5.js/blob/main/src/webgl/shaders/webgl2Compatibility.glsl to be either `texture2D` in WebGL1, or `texture` in WebGL2.

We need to do something similar for cubemap textures. In that file we'd have to `#define TEXTURE_CUBE` to either be `textureCube` in WebGL 1, or just `texture` in WebGL 2, and then use `TEXTURE_CUBE(...)` here.